app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    cmd, source_file_name, destination_path = command_list
+    destination_dir = os.path.dirname(destination_path)
+
+    if cmd != "mv" or len(command_list) < 2:
+        return
```

Comment:
If the command contain more than 3, it's incorrect too 
```suggestion
    if cmd != "mv" or len(command_list) == 3:
        return
```